### PR TITLE
[Misc] Add an example demonstrating how to run llumnix offline

### PR DESCRIPTION
**Pull request by KuilongCui**
1. update setup.py
2. add a example to run llumnix

### CODE REVIEW

File: examlpes/offline_inference.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+os.environ['HEAD_NODE_IP'] = '127.0.0.1'
+ray_cluster_port=37000
+
+# Note: launch_ray_cluster will stop current ray cluster first, then init a new one.
+launch_ray_cluster(ray_cluster_port=ray_cluster_port)
+connect_to_ray_cluster(port=ray_cluster_port)
+
+# Set manager args and engine args
+manager_args = EngineManagerArgs()
+engine_args = EngineArgs(model="/mnt/cuikuilong.ckl/Qwen-7B", worker_use_ray=True,
```
Comment (s5u13b) [inline examlpes/offline_inference.py:36]: Do not specify model through path /mnt/...,  use facebook opt？

Comment (KuilongCui) [reply examlpes/offline_inference.py]: done

File: llumnix/version.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```
Comment (ZeldaHuang) [inline llumnix/version.py:1]: License

Comment (KuilongCui) [reply llumnix/version.py:14]: done

**Review by ZeldaHuang — APPROVED** [review]

**Review by zhypku — APPROVED** [review]

**Pull request merged**